TACC/Camino
# task/FP-1120 & task/FP-1136: Allow standalone services (e.g. CMS/Docs/Portal)

Breaking up services into independent composable files to facilitate creating a fully customizable suite of services. Including default configurations for commonly deployed services

Updates to Makefile should make this change backward compatible for all currently deployed CORE portals except Frontera, which will need an update to the .env file to use the default compose configuration that includes docs

`BASE_COMPOSE_FILE=conf/camino/compose/docker-compose.core.docs.cms.yml`

**To Test Changes:** 
1. Clone the repo
2. Check out this branch
3. Create a `.env` file and populate with:
```
BASE_COMPOSE_FILE=docker-compose.nginx.yml
COMPOSE_FILE=
CAMINO_HOME=${PWD}
```
4. Run `make up`
5. Navigate to `localhost`
